scripts/workers/rl_submission_worker.py

```diff
@@ -164,9 +164,26 @@ def get_api_object(cluster_name, cluster_endpoint, challenge, evalai):
     return api_instance
 
 
-def get_core_v1_api_object(cluster_name, challenge, evalai):
+def get_api_client(cluster_name, cluster_endpoint, challenge, evalai):
```

Comment:
@Ayukha Can we please create a single function for getting all the three API client objects.

The input to the function will be `cluster_name, cluster_endpoint, challenge, evalai, api_client_type` and you will return the api_client based upon `api_client_type`

Reply:
Sure.

Reply:
@Ayukha Is this change done?
cc: @vkartik97 @rishabhbhardwaj